code/game/objects/items/implants/implantpad.dm

```diff
@@ -86,3 +102,60 @@
 	update_appearance(UPDATE_ICON)
 	update_static_data_for_all_viewers()
 	return TRUE
+
+/// Saves the currently inserted implant's deathrattle group.
+/obj/item/implantpad/proc/save_deathrattle_group(mob/user)
+	if(!inserted_case)
+		user.balloon_alert(user, "no case inside!")
+		return FALSE
+	if(!istype(inserted_case.imp, /obj/item/implant/deathrattle))
+		user.balloon_alert(user, "incompatible implant!")
+		return FALSE
+	var/obj/item/implant/deathrattle/inserted_implant = inserted_case.imp
+	var/datum/deathrattle_group/current_group = inserted_implant.current_group
+	if(!current_group)
+		user.balloon_alert(user, "no active group!")
+		return FALSE
+	saved_deathrattle_group = current_group
+	user.balloon_alert(user, "saved group [current_group.name]")
+	update_static_data_for_all_viewers()
+	return TRUE
+
+/// Sets the currently inserted implant's deathrattle group to saved.
+/obj/item/implantpad/proc/set_deathrattle_group(mob/user)
+	if(!inserted_case)
+		user.balloon_alert(user, "no case inside!")
+		return FALSE
+	if(!saved_deathrattle_group)
+		user.balloon_alert(user, "no saved deathrattle group!")
+		return FALSE
+	if(!istype(inserted_case.imp, /obj/item/implant/deathrattle))
+		user.balloon_alert(user, "incompatible implant!")
+		return FALSE
+	var/obj/item/implant/deathrattle/inserted_implant = inserted_case.imp
+	saved_deathrattle_group.register(inserted_implant)
+	user.balloon_alert(user, "registered to group [saved_deathrattle_group.name]")
+	inserted_case.name += " - [saved_deathrattle_group.name]"
```

Comment:
probably reset this to initial(name) + deathrattle group in case we're re-setting a previously-set implant?

maybe also add a check for deathrattle group types. perhaps if a less-restricted deathrattle shows up we don't want to set it to the wrong group, since deathrattle group decides alert sensitivity?